src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -2129,7 +2198,12 @@ export default class PlotsTab extends React.Component<IPlotsTabProps, {}> {
                 axisSelection.dataType !==
                     AlterationTypeConstants.MUTATION_EXTENDED &&
                 !this.props.store.molecularProfileIdToMolecularProfile
-                    .isComplete)
+                    .isComplete) ||
+            (axisSelection.dataType &&
```

Comment:
when would axisSelection not have a datatype?

Reply:
From the definition, I think the dataType can be undefined, so I add this checking
`export type AxisMenuSelection = {
    entrezGeneId?: number;
    genesetId?: string;
    genericAssayEntityId?: string;
    selectedGeneOption?: PlotsTabGeneOption;
    selectedDataSourceOption?: PlotsTabOption;
    selectedGenesetOption?: PlotsTabOption;
    selectedGenericAssayOption?: PlotsTabOption;
    dataType?: string;
    dataSourceId?: string;
    mutationCountBy: MutationCountBy;
    logScale: boolean;
};`

Reply:
ah right, thanks